packages/data-explorer/src/components/DataRowEdit.vue

```diff
@@ -0,0 +1,267 @@
+        <nav  v-if="parent" aria-label="breadcrumb">
+          <ol class="breadcrumb">
+            <li class="breadcrumb-item"><a href="#" v-on:click.prevent="showParent()">{{parent.dataTableLabel}} </a></li>
+            <li class="breadcrumb-item active" aria-current="page">{{dataTableLabel}}</li>
+          </ol>
+        </nav>
+        <h1>{{dataTableLabel}}</h1>
+
+        <form-component
+          id="data-row-edit-form"
+          :formFields="formFields"
+          :initialFormData="formData"
+          :formState="formState"
+          :options="formComponentOptions"
+          @valueChange="onValueChanged"
+          @addOptionRequest="onAddOptionRequest">
+        </form-component>
+
+        <div class="row">
+          <div class="col-md-12">
+            <button
+              id="cancel-btn"
+              @click.prevent="onCancelClick"
+              class="btn btn-secondary mr-1">
+              {{ 'data-row-edit-cancel-button-label' | i18n }}
+            </button>
+
+            <button
+              v-if="!isSaving"
+              id="save-btn"
+              class="btn btn-primary"
+              type="submit"
+              @click.prevent="onSubmit">
+              {{ 'data-row-edit-save-button-label' | i18n }}
+            </button>
+
+            <button
+              v-else
+              id="save-btn-saving"
+              class="btn btn-primary"
+              type="button"
+              disabled="disabled">
+              {{ 'data-row-edit-save-busy-state-label' | i18n }} <i
+              class="fa fa-spinner fa-spin " aria-hidden="true"></i>
+            </button>
+
+            <span v-if="!isSaving && formState.$invalid && formState.$touched && saveFailed"
+                  class="alert text-danger">
+                {{ 'data-row-edit-invalid-fields-msg' | i18n }}
+            </span>
+            <span v-else-if="alert && alert.type === 'danger' && alert.message"
+                  class="alert text-danger">
+                {{ alert.message }}
+            </span>
+
+          </div>
+        </div>
+
+      </div>
+      <div v-else class=""><i class="fa fa-spinner fa-spin fa-3x" aria-hidden="true"></i></div>
+
+    </div>
+
+    <div ref="refContainer"></div>
+
+  </div>
+
+</template>
+
+<script>
+import { FormComponent, EntityToFormMapper } from '@molgenis/molgenis-ui-form'
+import '../../node_modules/@molgenis/molgenis-ui-form/dist/static/css/molgenis-ui-form.css'
+import * as repository from '@/repository/dataRowRepository'
+import DataRowEdit from '@/components/DataRowEdit'
+import Vue from 'vue'
+
+export default {
+  name: 'DataRowEdit',
+  props: {
+    entity: {
+      type: String,
+      required: true
+    },
+    dataRowId: {
+      type: String,
+      required: false,
+      default: null
+    },
+    parent: {
+      type: Object,
+      required: false,
+      default: null
+    }
+  },
+  data () {
+    return {
+      dataExplorerBaseUrl: window.__INITIAL_STATE__.dataExplorerBaseUrl,
```

Comment:
is this in the context api?

Reply:
The 'dataExplorerBaseUrl' is actually unused in DataRowEdit. I removed the whole data property.